pkgs/development/python-modules/panel/default.nix

```diff
@@ -8,19 +8,39 @@
 , pyct
 , testpath
 , tqdm
+, callPackage
 }:
 
-buildPythonPackage rec {
+let
+  node = callPackage ./node {};
+in buildPythonPackage rec {
   pname = "panel";
-  version = "0.9.7";
-  # Version 10 attempts to download models from the web during build-time
-  # https://github.com/holoviz/panel/issues/1819
+  version = "0.11.1";
 
+  # Don't forget to also update the node packages
+  # 1. retrieve the package.json file
+  # 2. nix shell nixpkgs#nodePackages.node2nix
+  # 3. node2nix
```

Comment:
Maybe an update script for this should be written.

Reply:
Yes that would be a nice thing to have.